backend/generalhw.pm

```diff
@@ -106,20 +115,35 @@ sub relogin_vnc {
 sub do_start_vm {
     my ($self) = @_;
 
+    $self->poweroff_host;
```

Comment:
why poweroff on start?

Reply:
I want to make sure the board is off before starting the flash process.

Reply:
ok, can you add this as a comment in source please?

Reply:
Done.